tests/reporter/kubernetes.go

```diff
@@ -98,6 +98,7 @@ func (r *KubernetesReporter) JustAfterEach(specReport types.SpecReport) {
 	if r.artifactsDir == "" {
 		return
 	}
+	By("Collecting Logs for failed test")
```

Comment:
maybe "K8s Reporter collecting logs" ?
just so we won't use the word failed, as sometimes we search for error / failed as keyword